src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const obj of actions) {
+    switch (obj.type) {
+      case 'addProperties':
+        for (const key in obj.extraData) {
+          state[key] = obj.extraData[key];
+        };
```

Comment:
It is ok, but you should know that you can do the same this way:
`Object.assign(state, action.extraData);`